lib/Controller/PageController.php

```diff
@@ -302,6 +303,56 @@ public function index(string $token = '', string $callUser = '', string $passwor
 		return $response;
 	}
 
+	/**
+	 * @PublicPage
+	 * @NoCSRFRequired
+	 * @BruteForceProtection(action=talkRoomToken)
+	 *
+	 * @param string $token
+	 * @return TemplateResponse|NotFoundResponse
+	 */
+	public function recording(string $token = ''): Response {
```

Comment:
```suggestion
	public function recording(string $token): Response {
```

Doesn't need to be optional, since it's part of the URL already